dbt/include/databricks/macros/materializations/incremental/incremental.sql

```diff
@@ -2,7 +2,8 @@
 
   {#-- Validate early so we don't run SQL if the file_format + strategy combo is invalid --#}
   {%- set raw_file_format = config.get('file_format', default='delta') -%}
-  {%- set raw_strategy = config.get('incremental_strategy') or 'merge' -%}
+  {%- set raw_strategy = config.get('incremental_strategy', default='merge') -%}
```

Comment:
Could you revert this? `default` for `incremental_strategy` won't work. It should be `or 'merge'`.